build/ci_utils/src/programs/graal.rs

```diff
@@ -111,16 +110,17 @@ pub enum ComponentId {
     NodeJs,
     /// Graal.Python
     Python,
-    /// FastR
-    #[strum(serialize = "R")]
-    R,
```

Comment:
I have removed FastR for now, as I am unable to install it or to list it with `gu available`. I asked at [Truffle tools slack](https://graalvm.slack.com/archives/C016UG9L5UP/p1688574032797399). Until that is resolved, we will have to skip all the R polyglot tests. Tracked in ???